<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>juego_1</class>
 <widget class="QMainWindow" name="juego_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Capitales</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));

}</string>
   </property>
   <widget class="QWidget" name="panel_gen" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>621</width>
       <height>431</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="panel_jugador" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(206, 92, 0);
border-color: rgba(193, 125, 17, 0.6);
border-style:solid;
border-width:2px;
border-top-left-radius: 25px;
border-top-right-radius: 0px;
border-bottom-right-radius:25px;
border-bottom-left-radius: 0px; 
min-height: 70px; max-height:70px;
</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>599</width>
           <height>74</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 57 16pt &quot;Arial&quot;;
color: rgb(40,40,40); </string>
            </property>
            <property name="text">
             <string>Usuario:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="l_user">
            <property name="styleSheet">
             <string notr="true">font: 57 16pt &quot;Arial&quot;;
  color:  black; </string>
            </property>
            <property name="text">
             <string>usertest</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 57 16pt &quot;Arial&quot;;
color: rgb(40,40,40); </string>
            </property>
            <property name="text">
             <string>Puntaje:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="l_score">
            <property name="styleSheet">
             <string notr="true">font: 57 16pt &quot;Arial&quot;;
  color:  black; </string>
            </property>
            <property name="text">
             <string>41</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">font: 57 16pt &quot;Arial&quot;;
color: rgb(40,40,40); </string>
            </property>
            <property name="text">
             <string>Nivel:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="l_level">
            <property name="styleSheet">
             <string notr="true">font: 57 16pt &quot;Arial&quot;;
  color:  black; </string>
            </property>
            <property name="text">
             <string>level 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">QPushButton{ 
	border-color: rgba(193, 125, 17, 0.6);
border-style:solid;
border-width:2px;
border-top-left-radius: 25px;
border-top-right-radius: 0px;
border-bottom-right-radius:25px;
border-bottom-left-radius: 0px; 

	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(245, 121, 0, 255), stop:1 rgba(255, 255, 255, 255));
max-height: 32px;
min-height: 32px;
font-size: 20px;
color: rgb(40, 40, 40);
}

QPushButton:hover{
 
 
	border-color: rgb(233, 185, 110);
border-style:solid;
border-width: 3px; font-size: 14px;font-weight: bolder;
font-size:24px;
height: 64px;
background-image:url(:/background_3.jpg);
 color:  rgb(143, 89, 2);
} 
</string>
            </property>
            <property name="text">
             <string>Menu</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="l_continente">
        <property name="styleSheet">
         <string notr="true">font: 57 16pt &quot;Arial&quot;;
color: rgb(40,40,40);
min-height: 35px; max-height: 35px;</string>
        </property>
        <property name="text">
         <string>Continente</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="l_pregunta">
            <property name="styleSheet">
             <string notr="true">font: 57 17pt &quot;Arial&quot;;
color: rgb(40,40,40);
min-height: 50px;max-height: 50px;
 </string>
            </property>
            <property name="text">
             <string>Cual es la capital de</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="l_variante">
            <property name="styleSheet">
             <string notr="true">font: 57 17pt &quot;Arial&quot;;
color: rgb(40,40,40);
min-height: 50px;max-height: 50px;
 </string>
            </property>
            <property name="text">
             <string>Belgica?</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="imagen">
          <property name="styleSheet">
           <string notr="true">min-height: 150px; max-height: 150px;

 </string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="opc_1">
          <property name="styleSheet">
           <string notr="true">font: 57 16pt &quot;Arial&quot;;
color: rgb(40,40,40);  </string>
          </property>
          <property name="text">
           <string>Opcion 1</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="opc_2">
          <property name="styleSheet">
           <string notr="true">font: 57 16pt &quot;Arial&quot;;
color: rgb(40,40,40); </string>
          </property>
          <property name="text">
           <string>Opcion 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="opc_3">
          <property name="styleSheet">
           <string notr="true">font: 57 16pt &quot;Arial&quot;;
color: rgb(40,40,40); </string>
          </property>
          <property name="text">
           <string>Opcion 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_juego1">
          <property name="styleSheet">
           <string notr="true">QPushButton{ 
	border-color: rgba(193, 125, 17, 0.6);
border-style:solid;
border-width:2px;
border-top-left-radius: 25px;
border-top-right-radius: 0px;
border-bottom-right-radius:25px;
border-bottom-left-radius: 0px; 

	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(245, 121, 0, 255), stop:1 rgba(255, 255, 255, 255));
max-height: 32px;
min-height: 32px;
font-size: 20px;
color: rgb(40, 40, 40);
}

QPushButton:hover{
 
 
	border-color: rgb(233, 185, 110);
border-style:solid;
border-width: 3px; font-size: 14px;font-weight: bolder;
font-size:24px;
height: 64px;
background-image:url(:/background_3.jpg);
 color:  rgb(143, 89, 2);
} 
</string>
          </property>
          <property name="text">
           <string>Siguiente</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
